bots/source-queue-replicator/test/process.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
This should be in at `${systemdir}/config/leo-bus/process.js`

Reply:
I just put in dummy data